src/Surfnet/StepupGateway/SecondFactorOnlyBundle/Service/Gateway/AdfsService.php

```diff
@@ -77,15 +77,13 @@ public function handleAdfsRequest(LoggerInterface $logger, Request $httpRequest,
     }
 
     /**
-     * This method detectds if we need to return a ADFS response, If so ADFS parameters are returned.
```

Comment:
Did some boyscout work in this class while reading through the code path.

:evergreen_tree: :broom: